Makefile

```
@@ -6,7 +6,13 @@ local_no_proxy ?= ""
 
 name ?= powerfulseal
 version ?= `python setup.py --version`
-tag = $(name):$(version)
+arch1=$(shell uname -m)
+ifeq ($(arch1),x86_64)
+  arch2=amd64
+else ifeq ($(arch1),aarch64)
+  arch2=arm64
+endif
+tag = $(name):$(version)-$(arch2)
```

Thanks for making this change!

This is looking good, but for backwards compatibility I'd prefer to keep the naming scheme unchanged, and only append `-arm64` ad the end when necessary.

```suggestion
arch=$(shell uname -m)
ifeq ($(arch),x86_64)
  tag = $(name):$(version)
else ifeq ($(arch1),aarch64)
  tag = $(name):$(version)-arm64
endif
```